security.rst

```diff
@@ -2299,13 +2299,27 @@ the ``ROLE_SUPER_ADMIN`` permission:
     #[IsGranted('ROLE_ADMIN')]
     class AdminController extends AbstractController
     {
-        #[IsGranted('ROLE_SUPER_ADMIN')]
+        #[IsGranted('ROLE_SUPER_ADMIN', message: 'You are not allowed to access the admin dashboard.')]
         public function adminDashboard(): Response
         {
             // ...
         }
     }
 
+A custom status code can be set with the ``statusCode`` argument. This
+code will be used in the HTTP response::
+
+    // src/Controller/AdminController.php
+    // ...
+
+    use Symfony\Component\Security\Http\Attribute\IsGranted;
+
+    #[IsGranted('ROLE_ADMIN', statusCode: 403)]
```

Comment:
I think we can keep a custom statusCode here and specify in text above 403 is default value, seems confusing to me to explicitly define default value here

Reply:
You're right, updated!